pom.xml

```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>AmazonFresh</groupId>
+  <artifactId>AmazonFresh</artifactId>
```

Comment:
See standard rules for naming artifactId and make changes accordingly

Reply:
Done.